scripts/jenkins-yoctobuild-build.sh

```diff
@@ -505,15 +516,18 @@ then
             fi
         done
 
+
         prepare_and_set_PATH
         cd $WORKSPACE/meta-mender/tests/acceptance/
         mender-artifact write rootfs-image -t beaglebone -n test-update -u "$BUILDDIR"/tmp/deploy/images/beaglebone/core-image-base-beaglebone.ext4 -o successful_image_update.mender
         github_pull_request_status "pending" "Beaglebone acceptance tests started" "$BUILD_URL" "beaglebone_acceptance_tests"
         STATUS=0
         pytest --host=${SSH_TUNNEL_IP}:${BBB_PORT} --board-type=bbb || STATUS=$?
         if [[ $STATUS -eq 0 ]]; then
+            board_support_update "bbb" "passed"
             github_pull_request_status "success" "Beaglebone acceptance tests completed" "$BUILD_URL" "beaglebone_acceptance_tests"
         else
+            board_support_update "bbb" "failed"
```

Comment:
Can you call it `beaglebone` instead of `bbb`? The reason is that in the matrixed Jenkins I'm consolidating all of this into one function, so all we have access to is a variable with the name, and that name is `beaglebone`.